src/sql/src/plan/query.rs

```diff
@@ -5586,15 +5590,50 @@ impl Visit<'_, Aug> for WindowFuncCollector {
     }
 }
 
-/// Specifies how long a query will live. This impacts whether the query is
+/// Specifies how long a query will live.
+///
+/// Currently, it affects whether SHOW commands are allowed.
+///
+/// (This used to also impact whether the query is
 /// allowed to reason about the time at which it is running, e.g., by calling
-/// the `now()` function.
+/// the `now()` function. Nowadays, this is decided by a different mechanism.)
```

Comment:
Are you taking about `ExprPrepStyle` & friends? Maybe reference this struct here.

Reply:
Thanks, done